src/navigation/slam_gridmap/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>slam_gridmap</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="claudiakiss005@gmail.com">yurikun1033</maintainer>
+  <license>TODO: License declaration</license>
+
```

Comment:
Add dependencies for the msgs we're using - see other examples of our package's XMLs for how to do this